### ISSUE TITLE
[Bug]: QEMU 7.2.0-rc1 decompress error

### ISSUE THREAD

**Issue by PhotonNite** [ScoopInstaller/Main#4134]
### Prerequisites

- [X] I have written a descriptive issue title.
- [X] I have searched all issues/PRs to ensure it has not already been reported or fixed.
- [X] I have verified that I am using the latest version of Scoop and corresponding bucket.

### Package Name

QEMU

### Expected/Current Behaviour

Successfully installing/updating qemu
Fails to extract archive while installing/updating
[7zip.log](https://github.com/ScoopInstaller/Main/files/10034303/7zip.log)

### Steps to Reproduce

```console
PS C:\Users\***> scoop install qemu
Installing 'qemu' (7.2.0-rc1) [64bit] from main bucket
Starting download with aria2 ...
Download: Download Results:
Download: gid   |stat|avg speed  |path/URI
Download: ======+====+===========+=======================================================
Download: 76b368|OK  |    14MiB/s|C:/scoop/cache/qemu#7.2.0-rc1#https_qemu.weilnetz.de_w64_qemu-w64-setup-20221117.exe_dl.7z
Download: Status Legend:
Download: (OK):download completed.
Checking hash of qemu-w64-setup-20221117.exe ... ok.
Extracting dl.7z ... ERROR Exit code was 2!
Failed to extract files from C:\scoop\apps\qemu\7.2.0-rc1\dl.7z.
Log file:
  C:\scoop\apps\qemu\7.2.0-rc1\7zip.log

Please try again or create a new issue by using the following link and paste your console output:
https://github.com/ScoopInstaller/Main/issues/new?title=qemu%407.2.0-rc1%3a+decompress+error
```

### Possible Solution

I have no possible solution

### Scoop and Buckets Version

```console
Current Scoop version:
v0.3.1 - Released at 2022-11-15

'extras' bucket:
ce3b4585a (HEAD -> master, origin/master, origin/HEAD) ventoy: Update to version 1.0.82

'games' bucket:
ad4257a4 (HEAD -> master, origin/master, origin/HEAD) yuzu: Update to version 0-1237

'java' bucket:
aaa718ce (HEAD -> master, origin/master, origin/HEAD) temurin8-nightly-jre: Update to version 8.0.362-3.0.202211161814

'main' bucket:
3ec34fde2 (HEAD -> master, origin/master, origin/HEAD) ossutil: Add version 1.7.14 (#4132)

'nirsoft-alternative' bucket:
d736785 (HEAD -> master, origin/master, origin/HEAD) Assorted updates & new manifests

'nonportable' bucket:
1deabd2 (HEAD -> master, origin/master, origin/HEAD) nodejs-np: Update to version 19.1.0

'sysinternals' bucket:
e4f559d (HEAD -> main, origin/main, origin/HEAD) process-explorer: Fix hash
```

### Scoop Config

```console
aria2-enabled         : True
aria2-warning-enabled : False
last_update           : 17/11/2022 18:35:56
scoop_branch          : master
scoop_repo            : https://github.com/lukesampson/scoop
```

### PowerShell Version

```console
Name                           Value
----                           -----
PSVersion                      7.3.0
PSEdition                      Core
GitCommitId                    7.3.0
OS                             Microsoft Windows 10.0.19044
Platform                       Win32NT
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0…}
PSRemotingProtocolVersion      2.3
SerializationVersion           1.1.0.1
WSManStackVersion              3.0
```

**Comment by Zzyzx-Wolfe** [comment]
I am having a hash check failure when trying to update from 7.1.0 to 7.2.0-rc1. Possibly related?
```
scoop update *
Updating Scoop...
Updating 'extras' bucket...
 * 012cf1a5d sfsu: Update to version 1.1.1 (#9768)                       50 minutes ago
Updating 'main' bucket...
Scoop was updated successfully!
qemu: 7.1.0 -> 7.2.0-rc1
Updating one outdated app:
Updating 'qemu' (7.1.0 -> 7.2.0-rc1)
Downloading new version
qemu-w64-setup-20221117.exe (138.5 MB) [======================================================================] 100%
Checking hash of qemu-w64-setup-20221117.exe ... ERROR Hash check failed!
App:         main/qemu
URL:         https://qemu.weilnetz.de/w64/qemu-w64-setup-20221117.exe#/dl.7z
First bytes: 4D 5A 90 00 03 00 00 00
Expected:    5b5209da00805042184df1a2bd1a99d1a60f893d410b3124fd3b92313683043f43ecdd3b0bcd6a201e93a9f619a9f933747aa214f1e7cd047b5b9b481f911505
Actual:      68b517055affa887a0421ac37629460ee53f69dad79b0f6c5697c45e87358672ff356de7d3b739ce6389ff71e8ff61b92f3f9b46ed36c45a1d0e90a8cf13c118
```

**Comment by rashil2000** [comment]
Please try again with the latest fix: https://github.com/ScoopInstaller/Main/commit/e87529faf4602ac0ea385ef772e9f8117a3d73c3

**Comment by PhotonNite** [comment]
> Please try again with the latest fix: [e87529f](https://github.com/ScoopInstaller/Main/commit/e87529faf4602ac0ea385ef772e9f8117a3d73c3)

Works fine now
